RFC-005-SWAP-Basic-Bitcoin.adoc

```diff
@@ -125,6 +125,8 @@ Use the following table to fill in the correct parameters and build the actual H
 |20
 |===
 
+CAUTION: The `refund_timestamp` is of length 4 which means we https://github.com/bitcoin/bitcoin/blob/29082e8f40c360847882553ad1b3900e5e402688/src/script/script.h#L229-L246[have to use] the `OP_PUSHDATA4` op code to keep the script minimal. Data pushed using either `OP_PUSHDATA2` or `OP_PUSHDATA4` MUST be encoded in https://en.bitcoin.it/wiki/Script#Constants[*little-endian*]. Hence, you MUST convert `refund_timestamp` to *little-endian* before constructing the HTLC from its parameters.
```

Comment:
The link you posted states: `When used as numbers, byte vectors are interpreted as little-endian`, so I tried to find why pub-keys are not little-endian but couldn't find a reason. Just this: `Public keys (in scripts) are given as 04 <x> <y> where x and y are 32 byte big-endian integers representing the coordinates of a point on the curve or in compressed form given as <sign> <x> where <sign> is 0x02 if y is even and 0x03 if y is odd.`

which did not really help at all :S 

https://en.bitcoin.it/wiki/Protocol_documentation#Signatures

Reply:
I assumed that the OP codes just know how to interpret that data.